packages/eslint-plugin/docs/rules/consistent-return.mdx

```diff
@@ -9,8 +9,7 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/consistent-return** for documentation.
 
-This rule extends the base [`eslint/consistent-return`](https://eslint.org/docs/rules/consistent-return) rule.
-This version adds support for functions that return `void` or `Promise<void>`.
+It adds support for functions that return `void` or `Promise<void>`.
```

Comment:
I did a quick search and found that `class-methods-use-this` needs a similar change ([deploy preview link](https://deploy-preview-10734--typescript-eslint.netlify.app/rules/class-methods-use-this)), as it currently shows the `This rule extends the base ...` sentence twice:

> This rule extends the base [class-methods-use-this](https://eslint.org/docs/latest/rules/class-methods-use-this) rule from ESLint core. This rule extends the base [eslint/class-methods-use-this](https://eslint.org/docs/rules/class-methods-use-this) rule. It adds support for ignoring override methods or methods on classes that implement an interface.